back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/voting_results.rb

```diff
@@ -44,7 +44,7 @@ def mailer_class
     end
 
     def activity_triggers
-      { 'Notifications::VotingResults' => { 'created' => true } }
+      { 'Notifications::VotingResultsPublished' => { 'created' => true } }
     end
```

Comment:
Sebi discovered that we had missed updating this Notifications name in the past, so we're updating it now :)